Sources/SwiftKuery/Table.swift

```diff
@@ -395,12 +346,50 @@ open class Table: Buildable {
      var employeeTable = EmployeeTable()
      personTable = personTable.foreignKey(personTable.id, references: employeeTable.identifier)
      ```
-    
+
      - Parameter columns: A column that is the foreign key.
      - Parameter references: A column in the foreign table the foreign key references.
      - Returns: A new instance of `Table`.
     */
     public func foreignKey(_ column: Column, references: Column) -> Self {
         return foreignKey([column], references: [references])
     }
+
+    private static func columnsBelongTo(table: Table, columns: [Column]) -> Bool {
+        for column in columns {
+            if column.table._name != table._name {
+                return false
+            }
+        }
+        return true
+    }
+
+    /// Function that returns the SQL CREATE TABLE query as a String for this TABLE.
```

Comment:
Not really you but could you add a Usage example to the jazzy docs here since you understand this and are currently editing this section?